### PR TITLE
Update of readme

### PR DESCRIPTION
**Pull request by hollaa**
The text is based on the readme from our editor.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,23 @@
...
+
+## Run the project
+To run the project, double-click on the index.html file to open it in your default web browser. 
+
+## Style Guide
+
+The font used for the **comixcraft**-Showcase is _Nunito_, it is the default for everything.
+Any text in the comics themselves is in the font _Pangolin_.
+
+The colors, spacers, borders and other design elements are defined in our brand guide and only these should be used. They are the same as the ones in the editor.
```
Comment (Annacc211018) [inline README.md]: maybe add a link to where to find them in the editor repository

File: README.md [hunk 2]
```diff
@@ -1,9 +1,23 @@
...
+The font used for the **comixcraft**-Showcase is _Nunito_, it is the default for everything.
+Any text in the comics themselves is in the font _Pangolin_.
+
+The colors, spacers, borders and other design elements are defined in our brand guide and only these should be used. They are the same as the ones in the editor. 
 
 ## Hosting
 The content of this repository is hosted on https://showcase.comixcraft.com/ over [Netlify](https://www.netlify.com/).
 Updates on the website are done automatically on update of the `main`-branch.
 
 When adjusting the placeholder content, the deployment might need to be adjusted, depending on if a build process is necessary or not.
```
Comment (Annacc211018) [inline README.md]: can we remove this line?

File: README.md [hunk 3]
```diff
@@ -1,9 +1,23 @@
 # comixcraft - Showcase
 In this repository a showcase of the comixcraft project is saved.
-Currently it is just a placeholder-page. When implementing the placeholder can be replaced by any page, of any chosen technology.
+This website presents the comixcraft editor with its key functionalities and team members. 
+
+## Run the project
+To run the project, double-click on the index.html file to open it in your default web browser.
```
Comment (Annacc211018) [inline README.md]: or open it with a live server - at least that is what I needed to do

File: README.md (hunk 1 above)
Comment (esitsolf) [reply README.md]: You could also add the link to the fonts (https://github.com/comixcraft/Editor/tree/dev/public/fonts)

File: README.md [hunk 4]
```diff
@@ -1,9 +1,21 @@
...
-Currently it is just a placeholder-page. When implementing the placeholder can be replaced by any page, of any chosen technology.
+This website presents the comixcraft editor with its key functionalities and team members. 
+
+## Run the project
+To run the project, double-click on the index.html file to open it in your default web browser or open it with a live server. 
+
+## Style Guide
+
+The font used for the **comixcraft**-Showcase is _Nunito_, it is the default for everything.
+Any text in the comics themselves is in the font _Pangolin_.
```
Comment (esitsolf) [inline README.md]: You could also add the link to the fonts (https://github.com/comixcraft/Editor/tree/dev/public/fonts)

**Review by esitsolf — CHANGES_REQUESTED** [review]

**Pull request merged**